src/httpserver/http_request.hpp

```diff
@@ -33,19 +36,9 @@
 
 #include "httpserver/http_utils.hpp"
 
-struct MHD_Connection;
-
 namespace httpserver
 {
 
-class webserver;
-
-namespace http
-{
-    class header_comparator;
-    class arg_comparator;
-};
-
```

Comment:
Were these unused?

Reply:
For the first one; yes (kind of, friend declarations are them selves declarations so the explicit forward declaration isn't needed). For the rest, the relevant include is already there anyway.

Reply:
makes sense